src/Libuv/project.json

```diff
@@ -25,6 +25,7 @@
     }
   },
   "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.1-*",
```

Comment:
Isn't this package only used by Libuv internally? Most users wouldn't have to deal with it, right?